src/federation-info-plugin/utils/filterServicesInfo.js

```diff
@@ -0,0 +1,32 @@
+import filterDeep from 'deepdash/filterDeep'
+
+const filterServicesInfo = (services, query) => {
+  const filteredServices = []
+
+  if (query && services.length) {
+    services.map(service => {
```

Comment:
strange use of `.map` not doing any mapping (missing return and not assigned to any variable).

Suggest to either a `for (const service of services) {}`